pages/index.js

```diff
@@ -1,43 +1,53 @@
-// import external resources
-import useLocalStorageState from "use-local-storage-state";
+import { useState } from "react";
 import styled from "styled-components";
-// import internal resources
-import ProjectList from "@/components/ProjectList";
-import Logo from "@/components/svg/Logo.svg";
-import ProjectForm from "@/components/ProjectForm";
+import Logo from "@/public/Logo.svg";
 
-export default function HomePage({
-  addProject,
-  projects,
-  handleDeleteProject,
-}) {
+export default function HomePage({}) {
+  const [isClicked, setIsClicked] = useState(false);
+
+  function handleClick() {
+    setIsClicked(!isClicked);
+  }
   return (
     <>
-      <Header>
-        <Title>my projects</Title>
-      </Header>
-
-      <ProjectList
-        projects={projects}
-        handleDeleteProject={handleDeleteProject}
-      />
-
-      <ProjectForm addProject={addProject} />
+      <PageWrapper>
+        <Wrapper>
+          <Logo />
+          <a
+            href={`/ProjectPage`}
+            style={{ textDecoration: "none", color: "#94c3dd" }}
+          >
+            <StyledDiv onClick={handleClick} isClicked={isClicked}>
+              Welcome
+            </StyledDiv>
+          </a>
+        </Wrapper>
+      </PageWrapper>
     </>
   );
 }
+const PageWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #94c3dd;
+`;
 
-const Header = styled.header`
+const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
-  padding: 10px;
   align-items: center;
+  justify-content: center;
 `;
-
-const Title = styled.h1`
-  position: absolute;
-  top: 20px;
-  font-style: bold;
-  font-size: 17px;
-  color: #494b49;
+const StyledDiv = styled.div`
```

Comment:
As it is often regarded as not a good practice to nest a block element in an inline element I would not recommend to do this. And for your solution it is also not necessary. :)